seed/static/seed/js/controllers/data_upload_modal_controller.js

```diff
@@ -64,7 +64,12 @@ angular.module('BE.seed.controller.data_upload_modal', [])
       organization
     ) {
       $scope.cycles = cycles.cycles;
-      if ($scope.cycles.length) $scope.selectedCycle = $scope.cycles[0];
+      var cached_cycle = inventory_service.get_last_selected_cycle();
+      if ($scope.cycles.length && cached_cycle == null){
```

Comment:
When switching between Organizations, this logic still allows for a Cycle from a different organization to be selected.  It'll look like this:
<img width="548" alt="Screen Shot 2020-01-31 at 11 27 43 AM" src="https://user-images.githubusercontent.com/30608004/73565363-9d7acd80-441e-11ea-93ac-9c304377e195.png">
...which is similar to before except all the buttons are still clickable.

Reply:
I will follow the pattern linked above!